lib/jekyll/drops/static_file_drop.rb

```diff
@@ -0,0 +1,11 @@
+module Jekyll
+  module Drops
+    class StaticFileDrop < Drop
+      extend Forwardable
+      def_delegators :@obj, :name, :extname, :modified_time, :basename
+      def_delegator :@obj, :relative_path, :path
```

Comment:
I believe it will, based on https://github.com/jekyll/jekyll/blob/master/lib/jekyll/static_file.rb#L29, no?